<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropellerEnableWidget</class>
 <widget class="QWidget" name="PropellerEnableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>593</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>593</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="wReadyProp1" native="true">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>153</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QWidget" name="wReadyProp2" native="true">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>213</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QWidget" name="wReadyProp3" native="true">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>273</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QWidget" name="wReadyRudder1" native="true">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>332</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QWidget" name="wReadyRudder2" native="true">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>392</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QWidget" name="wRunRudder1" native="true">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>332</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QWidget" name="wRunRudder2" native="true">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>392</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QWidget" name="wRunProp3" native="true">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>273</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QWidget" name="wRunProp2" native="true">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>213</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QWidget" name="wRunProp1" native="true">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>153</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnableProp1">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>150</y>
     <width>54</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnableProp2">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>210</y>
     <width>54</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnableProp3">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>270</y>
     <width>54</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnableRudder1">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>329</y>
     <width>54</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnableRudder2">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>389</y>
     <width>54</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
